### DISCUSSION TITLE
What happens width #108?

### DISCUSSION THREAD

**Discussion by verberg** [cssbattle/forum#247] [Bugs reports]
Target 108: If I submit my old and 100% match solution, I get now only 34.8% match. What happens here?
Oh, it seems that all targets with empty variables no longer work! will it stay like this?
<img width="408" alt="Bildschirmfoto 2022-04-10 um 10 39 38" src="https://user-images.githubusercontent.com/94834403/162610081-bcf54db8-5b0b-46c1-90a1-b07822e25948.png">

**Comment by chinchang** [comment]
https://github.com/cssbattle/forum/discussions/248
